cmake/AddInputFileTests.cmake

```diff
@@ -98,12 +98,24 @@ function(add_input_file_tests INPUT_FILE_DIR)
     set(INPUT_FILE_CHECKS "parse;${INPUT_FILE_CHECKS}")
     list(REMOVE_DUPLICATES "INPUT_FILE_CHECKS")
 
+    # Read the timeout duration specified in input file, empty is accepted.
+    # The default duration is 2 seconds.
+    string(REGEX MATCH "#[ ]*Timeout:[^\n]+"
```

Comment:
Please make a quick separate PR that adds this timeout to input files to make this change visible to the other devs. (We discussed changes that should be stand-alone PRs recently in #1689)

Reply:
#1695

Reply:
#1695 got merged, and I've rebased this PR onto it.